opsdroid/core.py

```diff
@@ -114,15 +114,14 @@ def critical(self, error, code):
     @staticmethod
     def handle_async_exception(loop, context):
         """Handle exceptions from async coroutines."""
+        print("ERROR: Unhandled exception in opsdroid, exiting...")
```

Comment:
Why are we printing instead of logging?

Reply:
See my comment here: https://github.com/opsdroid/opsdroid/pull/1069#discussion_r327158672

The logger is in the process of getting cleaned up when the python process is exiting, so this was triggering that really long traceback from inside this handler.

Reply:
Probably just a mistake, I'll change it.

Reply:
I found it, it was this: https://bugs.python.org/issue26789